File: gm-utils/src/main/java/src/commom/utils/string/StringAfterLast.java
package src.commom.utils.string;

import src.commom.utils.integer.IntegerCompare;

public class StringAfterLast {

	private StringAfterLast() {}

	public static String get(String s, String substring) {
		int i = s.lastIndexOf(substring);
		if (IntegerCompare.eq(i, -1)) {
			return null;
		}
		return s.substring(i + substring.length());
	}

	public static String obrig(String s, String substring) {
		return StringObrig.get(get(s, substring));
	}

}

File: gm-utils/js/src/commom/utils/string/StringAfterLast.js
import IntegerCompare from '../integer/IntegerCompare';
import StringObrig from './StringObrig';

export default class StringAfterLast {

	static get(s, substring) {
		let i = s.lastIndexOf(substring);
		if (IntegerCompare.eq(i, -1)) {
			return null;
		}
		return s.substring(i + substring.length);
	}

	static obrig(s, substring) {
		return StringObrig.get(StringAfterLast.get(s, substring));
	}

}
